output/mapping/eforms/guidance.yaml

```diff
@@ -717,9 +717,11 @@
     - TED_EXPORT/FORM_SECTION/F25_2014/PROCEDURE/NOTICE_NUMBER_OJ
     TED guidance:
     - '[Reference a previous publication](../../operations/#reference-a-previous-publication)'
-    eForms guidance: '[Reference a previous publication](operations.md#reference-a-previous-publication)'
-    eForms example: <cac:NoticeDocumentReference><cbc:ID schemeName="notice-id-ref">123e4567-e89b-12d3-a456-426614174000-06</cbc:ID></cac:NoticeDocumentReference>
-    OCDS example: '{"relatedProcesses":[{"id":"1","relationship":["planning"],"scheme":"eu-oj","identifier":"123e4567-e89b-12d3-a456-426614174000-06"}]}'
+    eForms guidance: |-
+        1. Concatenate with [BT-1251-Lot Previous Planning Part Identifier ](#BT-1251-Lot) and [reference a previous publication](operations.md#reference-a-previous-publication).
+        1. Get the `RelatedProcess` created in the previous step and add the value of `ancestor::cac:ProcurementProjectLot/cbc:ID` to its `.relatedLots`.
+    eForms example: <cac:ProcurementProjectLot><cbc:ID schemeName="Lot">LOT-0001</cbc:ID><cac:TenderingProcess><cac:NoticeDocumentReference><cbc:ID schemeName="notice-id-ref">123e4567-e89b-12d3-a456-426614174000-06</cbc:ID><cbc:ReferencedDocumentInternalAddress>PAR-0001</cbc:ReferencedDocumentInternalAddress></cac:NoticeDocumentReference></cac:TenderingProcess></cac:ProcurementProjectLot>
+    OCDS example: '{"relatedProcesses":[{"id":"1","relationship":["planning"],"scheme":"eu-oj","identifier":"9c0fd704-64d3-4294-a3b6-6df45911ab9f-01-PAR-0001","relatedLots":["LOT-0001"]}]}'
```

Comment:
In both cases, should "concatenate with" be changed to "prepend to" or "append to", to ensure a consistent order?

Reply:
Resolving as the guidance no longer concatenates fields.